coalib/bearlib/aspects/__init__.py

```diff
@@ -1,111 +1,71 @@
+import re
+import sys
+from inspect import getmembers
+from types import ModuleType
+
 from .base import aspectbase
 from .meta import aspectclass
 from .taste import Taste, TasteError
 
-__all__ = ['Root', 'Taste', 'TasteError', 'aspectclass']
+from .root import Root
+from .Metadata import Metadata
+from .Redundancy import Redundancy
+from .Spelling import Spelling
```

Comment:
What are your opinion about changing this manual imports into an automatic function like https://github.com/coala/aspect-docs/blob/master/generate#L192 ?

I think it's better to make the "aspect indexing" become self-update-able if some other aspect module appear. Also coala use something like this to load their bears.